adoc/deployment-preparation.adoc

```diff
@@ -89,6 +89,16 @@ sudo zypper in -t pattern SUSE-CaaSP-Management
 This will install the `skuba` command line tool and `{tf}`; as well
 as various default configurations and examples.
 
+[NOTE] Using a Proxy Server with Authentication
+====
+Sometimes, you need a proxy server to be able to connects the SUSE Customer Center and the proxy server requires authentication. Create the file `/root/.curlrc` with the content:
+----
+--proxy https://PROXY_FQDN:PROXY_PORT
```

Comment:
Please help confirm proxy usage without problems.

https://curl.haxx.se/docs/manpage.html
https://www.suse.com/support/kb/doc/?id=7006845
https://github.com/SUSE/connect/blob/master/SUSEConnect.8.ronn#use-with-registration-proxy

Reply:
Not work...

Seems like right now SUSEConnect supports local SUSE registration proxy (RMT/SMT) only.

Reply:
SUSEConnect version `0.3.16`, but cannot find related tag on https://github.com/SUSE/connect

Reply:
Verify passed.

- server information with environment variable `http_proxy` with format `http_proxy=http://PROXY_IP_FQDN:PROXY_PORT`
- username/password in file `/root/.curlrc` with format `--proxy-user "username:password"`